Remove ip from logging, require nickname

zhyatt opened the pull request.

zhyatt committed:
Remove unhelpful IP address in favor of nickname only

argakiig commented on nmonit.py:
```diff
@@ -76,7 +76,7 @@ def _filter_host(connect:str) -> str:
-        address = f"{nickname} ({_filter_host(connect)})"
+        address = f"{nickname}"
     else:
         address = _filter_host(connect)
```
what about here as well? if a nickname is not provided it will still show the connection string host value

argakiig replied:
seems like perhaps the better thing to do would be to require nickname and enforce its usage instead

zhyatt replied:
Ah yeah, that makes sense. What is the best way to enforce that in this setup? Are you able to push that change here and I can review?

argakiig committed:
Update nmonit.py

remove default for nickname
update logic to always display nickname instead of parsing ip
removed now dead codepaths

argakiig commented:
pushed update, please verify working @zhyatt

argakiig committed:
Update nmonit.py

make nickname required

argakiig committed:
Update README.md

argakiig commented:
now requires nickname when passing flags during launch

zhyatt committed:
Fix invalid syntax
